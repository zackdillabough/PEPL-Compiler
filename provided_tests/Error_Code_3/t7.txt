POLY D = x + 2 x + 9 x + 5 x^2 + 10;
POLY B(a, b, c, d) = a^0 + b^1 + c^2 + d^3;
POLY C(e, f, g) = e^4 + 5 + f^6 + 7 + g^8 + 9;
START
INPUT P;
INPUT Q;
INPUT R;

B(P, Q, Q, R);

E(1);
F(10, 10, 27);

D(R);

A(A(
A(A(
A(A(
A(A(A(A(A(A(
A(A(





A(A(


A(A(

A(A(P, Q, R))))))))))
))))))))));




B(P, Q, Q, R);

E(1);
F(10, 10, 27);

D(R);

A(A(
A(A(
A(A(
A(A(A(A(A(A(
A(A(





A(A(


A(A(

A(A(P, Q, R))))))))))
))))))))));




B(P, Q, Q, R);

E(1);
F(10, 10, 27);

D(R);

A(A(
A(A(
A(A(
A(A(A(A(A(A(
A(A(





A(A(


A(A(

A(A(P, Q, R))))))))))
))))))))));



19 266 1 2 0 1



