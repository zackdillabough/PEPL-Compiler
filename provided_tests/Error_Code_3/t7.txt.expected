Error Code 3: 11 12 16 16 17 17 18 18 19 19 19 19 19 19 20 20 26 26 29 29 31 31 39 40 44 44 45 45 46 46 47 47 47 47 47 47 48 48 54 54 57 57 59 59 67 68 72 72 73 73 74 74 75 75 75 75 75 75 76 76 82 82 85 85 87 87 